site/docs/contribute.md

```diff
@@ -45,6 +45,17 @@ The Iceberg community prefers to receive contributions as [Github pull requests]
 * If a PR is related to an issue, adding `Closes #1234` in the PR description will automatically close the issue and helps keep the project clean
 * If a PR is posted for visibility and isn't necessarily ready for review or merging, be sure to convert the PR to a draft
 
+### Merging Pull Requests
+
+Most pull requests can be merged once a committer is satisfied with the code in the PR. For larger changes or additions to public
+APIs committers will wait at least 24 hours before merging to ensure there is no additional feedback from members of the community. As in
+all ASF governed projects committers are expected to act in the best interest of the project. If committer feels there might be a conflict
+of interest with a pull request they review, they are encouraged to ask for another committer to look at the pull request.
+
+There are several exceptions to this process:
+
+* Large changes or functional changes to a specification must go through the an [Iceberg improvement proposal](#apache-iceberg-improvement-proposals) before any code can be merged.
+* Changes to files under the `format` directory and `open-api/rest-catalog*` are considered specification changes. Unless already covered under an Iceberg improvement proposal, specification changes require their own vote (e.g. bug fixes or specification clarifications). The vote follows the ASF [code modification][apache-vote] model with three positive PMC votes required and no lazy consensus modifier. Grammar, spelling and minor formatting fixes are exempted from this rule.
```

Comment:
We also have `aws/src/main/resources/s3-signer-open-api.yaml`; would changes to that file fall under the "specification changes" category?

Reply:
I don't know.  Seems AWS specific based on the directory structure?  @jackye1995 ?

Reply:
yes, but that is not contributed by AWS. It was added to support remote signing feature in Tabular. I think it is more like a vendor integration feature, I would not consider that a part of the official REST spec. Thoughts? @nastra @danielcweeks